frontend/src/layouts/ProtectedLayout.vue

```diff
@@ -12,6 +12,8 @@ import LoginUserState from '@/store/FamLoginUserState';
 import { onMounted, ref, watch } from 'vue';
 import { useRoute } from 'vue-router';
 
+import SideNav from '@/components/common/SideNav.vue';
```

Comment:
Yeah, it wasn’t explicitly imported, and we have many similar cases throughout the app. It seemed strange because the components still work, but we lose the ability to see the prop types. From my research, it looks like this works because of Vite’s automatic import handling. However, it’s really bad practice since we lose type definition support.

Reply:
Got it, thanks!